invenio/modules/oaiharvester/config.py

```diff
@@ -21,6 +21,17 @@
 
 from __future__ import unicode_literals
 
+import os
+import sys
+
+
 OAIHARVESTER_DEFAULT_NAMESPACE_MAP = {
     None: "http://www.openarchives.org/OAI/2.0/",
 }
+"""The default namespace used when handling OAI-PMH results."""
+
+OAIHARVESTER_STORAGEDIR = os.path.join(sys.prefix, "var/data/oaiharvester/storage")
```

Comment:
We now have [CFG_DATADIR](https://github.com/inveniosoftware/invenio/blob/master/invenio/base/config.py#L93). It would also be best not to hardcode the slashes:

```python
OAIHARVESTER_STORAGEDIR = os.path.join(cfg['CFG_DATADIR'], 'oaiharvester', 'storage')
```

Naturally, `oaiharvester` could declare a base directory based on `CFG_DATADIR`, say,

```python
OAIHARVESTER_BASEDIR = os.path.join(cfg['CFG_DATADIR'], 'oaiharvester')
OAIHARVESTER_STORAGEDIR = os.path.join(OAIHARVESTER_BASEDIR, 'storage')
```